pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/model/PanDomainAuthSettings.scala

```diff
@@ -31,8 +32,16 @@ case class Google2FAGroupSettings(
 object PanDomainAuthSettings{
 
   def apply(settingMap: Map[String, String]): PanDomainAuthSettings = {
+    val cookieSettings = CookieSettings(
+      cookieName = settingMap("cookieName"),
```

Comment:
Are both these cookies still used?

Reply:
Unfortunately yes :(

There are still a couple of tools that need upgrading

Reply:
*facepalm

It might be nice to do that upgrade ahead of this so we can have v2 be just the new cookie (ideally with the spelling mistake fixed). Maybe that's too optimistic.

Reply:
Am I right in thinking that we only drop the assym cookie? Might be worth renaming these concepts to `legacyCookie`?

Reply:
We do still [drop the legacy cookie](https://github.com/guardian/pan-domain-authentication/blob/master/pan-domain-auth-play_2-6/src/main/scala/com/gu/pandomainauth/action/Actions.scala#L167). I have renamed the setting as suggested.

I will merge this PR, upgrade the last two remaining tools that still rely on the old cookie and then raise a follow up PR to remove the code just leaving the asymmetric one.